sdgym/synthesizers/base.py

```diff
@@ -55,6 +70,13 @@ def get_baselines(cls):
 
         return synthesizers
 
+    def _validate_modality_flag(self):
+        if not _is_valid_modality(self._MODALITY_FLAG):
+            raise ValueError(
+                f"The `_MODALITY_FLAG` '{self._MODALITY_FLAG}' of the synthesizer is not valid. "
+                "Must be either 'single_table' or 'multi_table'."
+            )
```

Comment:
This is not needed. The `_MODALITY_FLAG` is not something that we want to validate 'here', this validation should happen on a `metaclass` level, where if you are trying to do a subclass from the base it makes this validation, because this is not really 'user facing' in the way that you would pass it to the `method` of this class to validate it. 

PS: You can remove this validation for now.